parking_permits/tests/test_forms.py

```diff
@@ -92,6 +98,187 @@ def test_return_only_distinct_results_on_orders_with_multiple_permits_with_order
             self.assertEqual(name, qs[idx].permits.first().parking_zone.name)
 
 
+class PermitSearchFormTextSearch(TestCase):
+    def setUp(self):
+        self.address = AddressFactory(street_name="Pihlajakatu", street_number="23")
```

Comment:
`setUpTestData()` can be more efficient if the `address` attribute is not changed between tests, as it will run once.

Reply:
This `self.address` is actually not even needed, so it can be removed completely.